packages/react-components/react-skeleton/src/components/Skeleton/useSkeletonStyles.ts

```diff
@@ -1,33 +1,121 @@
-import { makeStyles, mergeClasses } from '@griffel/react';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
 import type { SkeletonSlots, SkeletonState } from './Skeleton.types';
 import type { SlotClassNames } from '@fluentui/react-utilities';
+import { tokens } from '@fluentui/react-theme';
+import { useFluent_unstable as useFluent } from '@fluentui/react-shared-contexts';
 
 export const skeletonClassNames: SlotClassNames<SkeletonSlots> = {
   root: 'fui-Skeleton',
-  // TODO: add class names for all slots on SkeletonSlots.
-  // Should be of the form `<slotName>: 'fui-Skeleton__<slotName>`
+};
+
+export const skeletonGroupClassName = 'fui-Skeleton-Group';
+
+const skeletonWaveAnimation = {
+  from: {
+    backgroundPositionX: '300%',
+  },
+  to: {
+    backgroundPositionX: '0%',
+  },
```

Comment:
```suggestion
   animationName: {
      from: {
        backgroundPositionX: '300% /* @noflip */',
      },
      to: {
        backgroundPositionX: '0% /* @noflip */',
      },
   }
```

Please use `@noflip` to avoid creating unused styles, [playground](https://griffel.js.org/try-it-out/?code=JYWwDg9gTgLgBAbziAhgawKYGUYE8A2GAzgDRxEAW0MFKAdgCZFwC%2BcAZlBCHAOQACAcyjB27DPgD0AY2gZeAbgBQSjAA9IsOAwzsUAV3zxUmHAWIAKBErhwuEGAC5EN2-VAoYwCHQByKEAxna1tQji4QYNcw2wAjFGk0YQh9RgAFCCJgLx8ADWdeAGYABmKAUjhJACo4fjoIdnxgMDgqyV4SaNCWTpi4GAgovriEpK5UhgysnLp8vnLKmrqGppa2jq7bHuiWV22WAEplIA) & [guidance](https://github.com/microsoft/fluentui/blob/master/rfcs/react-components/styles-handbook.md#avoid-unnecessary-rtl-transforms-by-using-noflip)

The same can be applied to `skeletonWaveAnimationRTL`